hyp3_gamma/rtc/rtc_sentinel.py

```diff
@@ -272,17 +276,23 @@ def rtc_sentinel_gamma(safe_dir: str, resolution: float = 30.0, radiometry: str
     os.mkdir(product_name)
     log_file = configure_log_file(f'{product_name}/{product_name}.log')
     log_parameters(safe_dir, resolution, radiometry, scale, speckle_filter, dem_matching, include_dem, include_inc_map,
-                   include_scattering_area, include_rgb, orbit_file, product_name)
+                   include_scattering_area, include_rgb, orbit_file, product_name, dem_name)
 
     log.info('Preparing DEM')
-    dem_type = 'COP30'
+    dem_tif = 'dem.tif'
     dem_image = 'dem.image'
     dem_par = 'dem.par'
-    geometry = get_geometry_from_kml(f'{safe_dir}/preview/map-overlay.kml')
-    with NamedTemporaryFile() as dem_tif:
-        prepare_dem_geotiff(dem_tif.name, geometry)
-        run(f'dem_import {dem_tif.name} {dem_image} {dem_par} - - $DIFF_HOME/scripts/egm2008-5.dem '
+    if dem_name == 'copernicus':
+        dem_type = 'COP30'
```

Comment:
Since this is only used for `hyp3_metadata`, we should probably just update it to use `copernicus`, though we may need to specify a `dem_resolution` in the future...

No matter for this PR though

Reply:
And with this morning's round of hyp3-metadata-templates edits, the name value is only displayed in the README.md.txt:

```
The DEM used for this product is <dem_name>, which has a native pixel spacing of 1 arc second (about 30 meters).

The Copernicus DEM GLO-30 is a global Digital Surface Model (DSM) derived from the WorldDEM. The WorldDEM is based on radar satellite data acquired by the TanDEM-X mission, which was funded by the German Aerospace Center (DLR) and Airbus Defence and Space, and edited to flatten water bodies, provide consistent flow of rivers, and apply corrections to shore/coastlines and special features. For an overview of the dataset, visit https://spacedata.copernicus.eu/fr/dataset-details?articleId=394198. The product handbook is available at https://spacedata.copernicus.eu/documents/20126/0/GEO1988-CopernicusDEM-SPE-002_ProductHandbook_I1.00.pdf.
```

Heidi and I were thinking `GLO-30` would be the best thing to put there, since `Copernicus DEM GLO-30` appears in the next sentence.  But that'll be a subsequent hyp3-gamma PR after the next hyp3-metadata-templates release.

Reply:
makes sense to me; `dem_name` can just be `GLO-30` then I think